.github/dependabot.yml

```
@@ -4,3 +4,9 @@ updates:
     directory: /
     schedule:
       interval: "weekly"
+        target-branch: "main"
+  - package-ecosystem: "github-actions"
+    directory: /
+    schedule:
+      interval: "weekly"
+        target-branch: "ci-images"
```

`target-branch` should be at the same level as `schedule`
```suggestion
    target-branch: "ci-images"
```

(Same for line 7)